devlib/module/gem5stats.py

```diff
@@ -66,6 +67,7 @@ def __init__(self, target):
         self._stats_file_path = os.path.join(target.platform.gem5_out_dir,
                                             'stats.txt')
         self.rois = {}
+        self._dump_pos_cache = {0: 0}
```

Comment:
I don't think python provides that by default. This is what we need:
https://github.com/grantjenks/sorted_containers
Not sure if it's a good idea to add external dependencies to devlib ...